dev-util/bugbite-cli/bugbite-cli-0.0.12-r1.ebuild

```diff
@@ -28,14 +29,15 @@ pkg_setup() {
 		export RUSTFLAGS="-C target-feature=+crt-static ${RUSTFLAGS}"
 		export static_stuff="--target ${rust_target}"
 	fi
+	rust_pkg_setup
```

Comment:
I don't think we should call `rust_pkg_setup` _after_ some code which calls/checks `rustc` (look above it).